packages/itwinui-react/package.json

```diff
@@ -99,20 +100,18 @@
     "@types/react-transition-group": "^4.4.10",
     "@typescript-eslint/eslint-plugin": "^5.60.0",
     "@typescript-eslint/parser": "^5.60.0",
+    "@vitest/coverage-v8": "^1.2.1",
     "concurrently": "^5.3.0",
     "eslint": "^8.43.0",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-require-extensions": "^0.1.3",
-    "jest": "^29.0.0",
-    "jest-cli": "^29.0.0",
-    "jest-environment-jsdom": "^29.0.0",
-    "jest-junit": "^13.2.0",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "ts-jest": "^29.0.0",
     "ts-node": "^10.0.0",
```

Comment:
Some more dependencies can be removed: `ts-jest`, `ts-node`, `@types/jest`

Also you might want to use the latest version of [`@testing-library/jest-dom`](https://github.com/testing-library/jest-dom), which has support for `vitest`.